host/sgx/create.c

```diff
@@ -460,6 +463,13 @@ static oe_result_t _configure_enclave(
                     enclave, max_host_workers, max_enclave_workers));
                 break;
             }
+#ifdef OE_WITH_EXPERIMENTAL_EEID
+            case OE_EXTENDED_ENCLAVE_INITIALIZATION_DATA:
+            {
+                // Nothing
```

Comment:
If EEID is a setting, shouldn't enabling it do something?

Reply:
Not during `_configure_enclave`. What type of thing should it do here? This is just to avoid falling through into the invalid parameter.